cmd/thanos/receive.go

```diff
@@ -305,49 +301,35 @@ func runReceive(
 			return errors.Wrap(err, "setup gRPC server")
 		}
 
-		mts := store.NewProxyStore(
-			logger,
-			reg,
-			dbs.TSDBLocalClients,
-			comp,
-			labels.Labels{},
-			0,
-			store.LazyRetrieval,
-		)
+		//new change
```

Comment:
@Kartik-Garg Can you pls explain this change a bit in the issue description?

Reply:
@samkulkarni20 This change is outdated now :p , so resolving it now